### ISSUE TITLE
User is assigned Letter in firebase, but assignments are not showing up. 

### ISSUE THREAD

**Issue by ctflores** [yeatmanlab/roar#59]
I assigned a set of test users to an administration that was assigned Letter, including username `Ascend44`. You can see from the screenshot that the user is in the administration and has been assigned Letter.
![image](https://github.com/yeatmanlab/roar/assets/35785147/03097ea4-78c8-49f0-b96a-831d2fe63edf)

However, when I went to log into roar.education as `Ascend44`, the user appears to have no assignments. I went through the steps Adam shared to empty the cache and do a hard reload, and the user still appears to have no assignments. 

![image](https://github.com/yeatmanlab/roar/assets/35785147/7dc5cf43-0b37-406b-b562-6af90bbe24fb)

This is the first time I have ever seen this issue. It affected other test users uploaded at the same time to the same administration (e.g. `Ascend42`). However, it did not affect all of them. Some work just fine (e.g. `Ascend41`). 

For what it's worth, `Ascend41` can see the Letter game and has no errors in the console. `Ascend1`, another test user from the same upload, appears to be able to see Letter, but has similar looking errors in the console. 
![image](https://github.com/yeatmanlab/roar/assets/35785147/92eb4ad3-5cb7-4c11-b706-0df2cf103484)

**Comment by richford** [comment]
I noticed that the `dateOpened` field is Sep 13 at 7:45 pm. Did all of the issues you encountered happen after that time?

**Comment by ctflores** [comment]
Yes

**Comment by ctflores** [comment]
It may have been a buggy upload. I didn't receive and FirebaseErrors upon upload, but I did notice the dashboard looked weird after I uploaded. I just refreshed the page and it looked fine, so I didn't connect the two errors. Maybe they're related? If you want to just delete all of the users that start with "Ascend" and I can re-upload them, that works for me. I just want to make sure it's not indicative of something that affects other csv uploaded users.
